etc/profile.d/baseimage.sh

```diff
@@ -7,3 +7,10 @@ fi
 alias pip="pip3 --no-cache-dir"
 alias pip3="pip3 --no-cache-dir"
 alias python="python3"
+
+# Flask
+export FLASK_APP="application.py"
+
+# PATH
+export PYENV_ROOT=/opt/pyenv
+export PATH=/opt/cs50/bin:/opt/bin:/usr/local/sbin:/usr/local/bin:"$PYENV_ROOT"/shims:"$PYENV_ROOT"/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
Wouldn't quoting the whole value be sufficient as opposed to quoting each `PYENV_ROOT` separately?

Reply:
Yup but feels sloppy. Most folks seem to just quote vars themselves. Cf. http://tldp.org/LDP/abs/html/quotingvar.html.

> When referencing a variable, it is generally advisable to enclose its name in double quotes.